services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    sessions = MovieSession.objects.all()
+    if session_date:
+        sessions = sessions.filter(
+            show_time__date=datetime.strptime(session_date, "%Y-%m-%d")
+        )
+    return sessions
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> MovieSession:
+    movie_session = MovieSession.objects.get(id=session_id)
+    if show_time:
+        movie_session.show_time = show_time
+    if movie_id:
+        movie_session.movie_id = movie_id
+    if cinema_hall_id:
+        movie_session.cinema_hall_id = cinema_hall_id
+    movie_session.save()
+
+    return movie_session
+
+
+def delete_movie_session_by_id(session_id: int) -> None:
+    return MovieSession.objects.get(id=session_id).delete()
```

Comment:
Here you could reuse `get_movie_session_by_id` function to make code DRY
Also no need to return result from deletion